app/src/main/java/com/android/sample/MainActivity.kt

```diff
@@ -53,7 +54,7 @@ fun PlateSwipeApp() {
 
   val userViewModel = UserViewModel.Factory.create(UserViewModel::class.java)
 
-  NavHost(navController = navController, startDestination = Route.AUTH) {
+  NavHost(navController = navController, startDestination = Route.SWIPE) {
```

Comment:
Correct me if I'm wrong but shouldn't this line be:
```suggestion
 `NavHost(navController = navController, startDestination = Route.AUTH) {
```
 ?

Reply:
Yes, I modified it for testing and forgot to switch it back.